runtime/integration-tests/src/xcm_transfers.rs

```diff
@@ -31,6 +34,59 @@ use imbue_kusama_runtime::{
 use orml_traits::MultiCurrency;
 use pallet_proposals::traits::RefundHandler;
 
+#[test]
+fn transfer_treasury_to_parachain_grant_escrow_address() {
+    let transfer_amount: Balance = ksm_amount(1);
+    let treasury_origin = TreasuryOrigin::Kusama;
+    let kusama_treasury_address =
+        <R as pallet_proposals::Config>::RefundHandler::get_treasury_account_id(treasury_origin)
```

Comment:
Perhaps we should have a treasury interface type for handling these interactions

Reply:
Good idea. Would make a good starting task 

Raise an issue please.